package-lock.json

```diff
@@ -5,7 +5,6 @@
   "requires": true,
   "packages": {
     "": {
-      "name": "to-words",
```

Comment:
Please help with more details on changes in the package-lock as it's unclear.

Reply:
I took the update from the main branch for vulnerabilities - the pull updated versions for certain modules

Reply:
I'll suggest you to remove package-lock from your PR. I periodically take care of maintaining the package updates.

Reply:
ok, let me remove and update the PR.